src/components/Form.js

```diff
@@ -102,20 +102,20 @@ class Form extends React.Component {
                 name="zipcode"
               />
 
-              <Select
+              <Input
                 animate={animate}
-                form="registration"
                 label="State"
-                menuType="states"
+                menuType={STATES}
```

Comment:
From looking at just this code, it's not obvious to me what the menuType prop does, so I think we would do well to rename it to something else that is more descriptive.  Maybe `menuData` or `dropdownData`.